tests/main/security-private-tmp/task.yaml

```diff
@@ -8,25 +8,26 @@ environment:
 
 prepare: |
     echo "Given a basic snap is installed"
-    . $TESTSLIB/snaps.sh
+    . "$TESTSLIB/snaps.sh"
     install_local test-snapd-tools
 
     echo "And another basic snap is installed"
-    mkdir -p $SNAP_INSTALL_DIR
-    cp -ra $TESTSLIB/snaps/test-snapd-tools/* $SNAP_INSTALL_DIR
-    sed -i 's/test-snapd-tools/not-test-snapd-tools/g' $SNAP_INSTALL_DIR/meta/snap.yaml
-    snapbuild $SNAP_INSTALL_DIR .
+    mkdir -p "$SNAP_INSTALL_DIR"
+    cp -ra "$TESTSLIB"/snaps/test-snapd-tools/* "$SNAP_INSTALL_DIR"
+    sed -i 's/test-snapd-tools/not-test-snapd-tools/g' "$SNAP_INSTALL_DIR/meta/snap.yaml"
+    snapbuild "$SNAP_INSTALL_DIR" .
```

Comment:
Same! Sorry for bothering, but this is a pretty large diff, and those quotings are a significant part of it, without any real advantages for us.